### DISCUSSION TITLE
ability to send whatsapp 360dialog template message via Messages API

### DISCUSSION THREAD

**Discussion by voko3456** [chatwoot/chatwoot#3302] [Feature Requests]
right now if we send template message via 360dialog api it will no show in chat history we need to send WhatsApp 360dialog template message via Messages API then it can show in chat history

**Comment by sojan-official** [comment]
If you send the messages directly via 360dialog API there is no way for chatwoot to know that these messages were sent. Instead, use the chatwoot messages API so that the record remains in chatwoot. Internally chatwoot will use the 360 dialog APIs to deliver the message to whatsapp. 

Chatwoot Messages API also support the template messages, provided that the message payload is same as the template.

**Reply by voko3456** [reply]
But we are sending interactive messages template and is chatwoot messages api support to send interactive message template with variables?

**Reply by sojan-official** [reply]
we don't support interactive message templates at the moment. But we have it in our roadmap.
